ui/components/ui/multi-select.tsx

```
@@ -0,0 +1,1066 @@
+                  'multiselect-scrollbar max-h-[40vh] overflow-y-auto',
+                  screenSize === 'mobile' && 'max-h-[50vh]',
+                  'overscroll-behavior-y-contain',
+                )}
+              >
+                <CommandEmpty>{emptyIndicator || 'No results found.'}</CommandEmpty>{' '}
+                {!hideSelectAll && !searchValue && (
+                  <CommandGroup>
+                    <CommandItem
+                      key="all"
+                      onSelect={toggleAll}
+                      role="option"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<option>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)